app/build.gradle

```diff
@@ -145,6 +145,7 @@ buildscript {
     ext.firebase_version = '17.2.3'
     ext.crashlytics_version = '2.10.1'
     ext.mercadopago_version = '4.40.1'
+    ext.lottie_version = '3.4.0'
```

Comment:
Is there a version conflict with the [last one](https://github.com/airbnb/lottie-android/releases/tag/v4.1.0)?

Reply:
Apparently, android studio suggests me that version